scripts/airdrop.js

```diff
@@ -21,12 +23,56 @@ const argv = default_yargs
     describe: "Do not actually send transactions, just simulate their submission",
   }).argv
 
+const getFileExtension = function (filename) {
```

Comment:
This method is part of the NodeJS standard packages: https://nodejs.org/dist/latest-v12.x/docs/api/path.html#path_path_extname_path